android/KMEA/app/src/main/java/com/tavultesoft/kmea/util/FileUtils.java

```diff
@@ -239,14 +239,45 @@ public static boolean isCustomKeyboard(String u) {
     if (u == null) {
       return ret;
     }
+    String lowerU = u.toLowerCase();
     Pattern pattern = Pattern.compile("^http(s)?://(.+\\.)?keyman.com/.*");
-    Matcher matcher = pattern.matcher(u);
+    Matcher matcher = pattern.matcher(lowerU);
     if (matcher.matches()) {
       ret = false;
     }
     return ret;
   }
 
+  /**
+   * Utility to parse a URL and determine if it's a keyman:<method>
+   * Currently, primarily matching for "keyman://" or "keyman:download?"
```

Comment:
Why are you still supporting `keyman://` here?

Reply:
The Android intent handler triggers on both `keyman://` and `keyman:download`.
I've blocked `keyman://` in FileUtils and MainActivity.